dev_guide/persistent_volumes.adoc

```diff
@@ -16,34 +16,13 @@ Storage is provisioned by an administrator by creating `*PersistentVolume*`
 objects from sources such as GCE Persistent Disks, AWS Elastic Block Stores
 (EBS), and NFS mounts.
 
-ifdef::openshift-enterprise[]
 [NOTE]
 ====
-Persistent volume plug-ins other than the supported
-link:../install_config/persistent_storage/persistent_storage_nfs.html[NFS] plug-in, such as
-link:../rest_api/kubernetes_v1.html#v1-awselasticblockstorevolumesource[AWS
-Elastic Block Stores (EBS)],
-link:../rest_api/kubernetes_v1.html#v1-gcepersistentdiskvolumesource[GCE
-Persistent Disks],
-link:../rest_api/kubernetes_v1.html#v1-glusterfsvolumesource[GlusterFS],
-link:../rest_api/kubernetes_v1.html#v1-iscsivolumesource[iSCSI], and
-link:../rest_api/kubernetes_v1.html#v1-rbdvolumesource[RADOS (Ceph)], are
-currently in
-link:../whats_new/ose_3_0_release_notes.html#technology-preview[Technology
-Preview]. The Administrator Guide provides instructions on provisioning an
-OpenShift cluster with
-link:../install_config/persistent_storage/persistent_storage_nfs.html[persistent storage using NFS].
+The Installation and Configuration Guide provides instructions on provisioning
+an OpenShift cluster with
+link:../install_config/persistent_storage/persistent_storage_nfs.html[persistent
+storage using NFS].
```

Comment:
We don't need to specifically call out NFS anymore, since we have multiple admin topics on various volume plug-ins now.